cmd/run.go

```diff
@@ -54,6 +54,12 @@ Run sidecar only:
 		viper.BindPFlag("placement-host", cmd.Flags().Lookup("placement-host"))
 	},
 	Run: func(cmd *cobra.Command, args []string) {
+		if configFile == "" {
```

Comment:
This is checked here and not added in defaults so that the automatic docs generated do not have any platform dependent defaults.

Reply:
https://github.com/dapr/cli/runs/1009642534?check_suite_focus=true#step:8:28